src/api.ts

```diff
@@ -152,12 +152,20 @@ export interface TestResults {
   creationTime: string;
 }
 
+export type ResultInterpretationTheme = 'POSITIVE' | 'NEGATIVE' | 'NEUTRAL' | 'MUTED';
+
+export interface ResultInterpretation {
+  name: string;
+  theme: ResultInterpretationTheme;
+}
+
 export interface Test {
   id: string;
   userId: string;
-  testTypeId: string;
+  testType: TestType;
   creationTime: string;
   results?: TestResults;
+  resultsInterpretations?: ResultInterpretation[];
```

Comment:
When is it not going to be present?

Reply:
When results aren't present. We've separated creating tests and adding results (for certain workflows), even though we don't use it in the ui currrently.